onboarding/mlsys/1-gradients.md

```diff
@@ -210,7 +229,9 @@ Additionally, in the "select one or more `(x, ytrue)` input pairs" step, for
 now you should just select the next `(x, ytrue)` pair in the dataset, and it
 should wrap back to the beginning of the dataset after reaching the end.  So if
 your dataset is of length 10, then your eleventh iteration should reuse the
-first `(x, ytrue)` pair of your dataset.
+first `(x, ytrue)` pair of your dataset.  Effectively this will let you loop
+seamlessly through multiple epochs your dataset, a strategy that is common in
+large-scale machine learning training.
```

Comment:
This seems like a fine thing to add but it isn't really what I intended. What I was more going for was an explanation of "why aren't we just using all the data each iteration?" since there's only like 4 datapoints in our examples

Reply:
Got it.  That makes sense, especially after the `select one or more` phrasing in the algorithm description.